yasser_sharabati/task_manager.py

```diff
@@ -0,0 +1,99 @@
+from task import Task, UrgentTask
+from utils import validate_date, get_valid_input, validate_status, validate_priority, save_tasks_to_file, load_tasks_from_file
+
+tasks, urgent_tasks = load_tasks_from_file()
+
+def add_task():
+    title = get_valid_input("Enter task title: ")
```

Comment:
Please allow the user to enter the value first, then perform the validation.

Do not keep the input value within the validation method